To-Do-app/src/Components/TodoTaskList.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { Dispatch, SetStateAction } from 'react'
+import { Grid } from '@material-ui/core'
+import { ITask } from '../Interface'
+import TodoTask from './TodoTask'
+interface Props {
+    todoList: ITask[]
+    setTodoList: Dispatch<SetStateAction<ITask[]>>
+}
+
+function TodoTaskList({ todoList, setTodoList }: Props) {
+
+    const deleteTask = (taskNameToDelete: number): void => {
```

Comment:
the variable name says "Name" but the type is number

Reply:
I'll change the variable name